linker/Linker.Steps/MarkStep.cs

```diff
@@ -1401,6 +1403,20 @@ static bool IsMulticastDelegate (TypeDefinition td)
 			return td.BaseType != null && td.BaseType.FullName == "System.MulticastDelegate";
 		}
 
+		protected virtual bool AlwaysMarkTypeAsInstantiated (TypeDefinition td)
+		{
+			switch (td.FullName) {
```

Comment:
FullName is expensive property, use name + namespace instead

Reply:
Fixed